Sources/Commands/show-dependencies.swift

```diff
@@ -107,7 +107,13 @@ private final class JSONDumper: DependenciesDumper {
                 ])
         }
 
-        print(convert(rootpkg).toString(prettyPrint: true))
+        print(convert(rootpkg).toString(prettyPrint: true, sortingKeysBy: sortingKeysBy(a:b:)))
+    }
+
+    /// To bring `dependencies` key to the last order.
+    /// See: https://bugs.swift.org/browse/SR-5624
+    func sortingKeysBy(a: String, b: String) -> Bool {
```

Comment:
Again, this should ideally not be a member function; nothing about it has anything to do with a particular instance of `JSONDumper`.

The way I see it, it can be moved above into `dump(dependenciesOf:)`, either as a closure or as its own function (like `convert(_:)` above). If you're going to choose to have it as its own function, please use a more descriptive name than just `sortingKeysBy` (something like `areInIncreasingOrderExceptDependenciesLast`--except maybe better; the name should generally begin with `is` or `are` because it returns a `Bool`).

But in either case, it should not be a member function of `JSONDumper`.